cross-reference/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Use an official Python runtime as a parent image
+FROM python:3.12.3-alpine
+
+# Set environment variables
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+# Set work directory
+WORKDIR /app
+
+# Install system dependencies
+RUN apk update \
+    && apk add --no-cache \
+        gcc \
+        libc-dev \
+        mariadb-dev \
+    && rm -rf /var/cache/apk/*
+
+# Install python dependencies
+COPY requirements.txt /app
+RUN pip install --no-cache-dir --upgrade pip && \
+    pip install --no-cache-dir -r requirements.txt
+
+# Copy project
+COPY ./crossreference /app/
```

Comment:
It's probably cleaner to use this as a volume mount with the generated image. It would avoid regenerating the image anytime we change app code source...

Reply:
I think it's easier to have all contained in the image, for local Development purposes.
If this sounds sane, then also making use of `DEV_ *container tags` will help separate the two environments.

Reply:
Why is that? Can't you have the content locally the same way? Also it's against containers best practice to do that. I am really not sure about the benefit of this approach.